packages/connectivity/src/index.ts

```diff
@@ -1,6 +1,5 @@
-/* eslint-disable tsdoc/syntax */
 /**
- * [[include:connectivity/README.md]]
+ * [[include:connectivity/README.md]].
```

Comment:
There should not be any fullstop here. Can we add an exception?

Reply:
Ok, I wasn't aware of it. I'll look into it.

Reply:
I turned the `require-description-complete-sentence` rule off for index.ts file in `.eslintrc.js`